.github/workflows/accessibility-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Run accessibility tests
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Question: Is the timezone by default UTC?

Reply:
Yes, GHA seems to run on UTC by default: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule